### PR TITLE
GraphQLConf 2023 recap

### PR DESCRIPTION
**Pull request by kamilkisiela**

### CODE REVIEW

File: website/pages/blog/graphqlconf-2023-recap.mdx [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+being announced at the conference and how we all can use them to improve our GraphQL developer
+experience. This talk also acted as a guide to some of the talks at the conference. He covered
+important releases for GraphQL Clients and GraphQL Gateways. One of the key takeaways was the
+emergence of GraphQL gateways as a common pattern.
+
+For a long time, developers had limited options when it came to choosing a GraphQL gateway. However,
+now we are **witnessing a flourishing and diverse range of gateways being built in various
+languages** like Apollo Router, Hot Chocolate, Cosmos, and our own GraphQL Mesh. The future promises
+friendly competition between the teams and companies behind these projects. Uri highlighted the
+available choices and alternatives out there and also talked about how should you go about choosing
```
Comment (lirona) [inline website/pages/blog/graphqlconf-2023-recap.mdx:86]:
```suggestion
available choices and alternatives out there and also talked about how you should go about choosing
```

File: website/pages/blog/graphqlconf-2023-recap.mdx [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+He then announced **the most important release of the conference - the new GraphQL Fusion spec** by
+ChilliCream and The Guild, Hasura, IBM, AWS AppSync, WunderGraph and many more. An open spec, under
+the Foundation, for remote execution and federation of services that challenges the existing
+patterns of specification-less Apollo Federation with a fresh approach.
+
+He ended with the message - We are just getting started!
+
+**The conference was a testament to the growing popularity and importance of GraphQL.** Whether
+attendees were just starting to explore GraphQL, had already adopted it, or were active contributors
+in the GraphQL ecosystem, everyone played a vital role in the success and future of GraphQL. The
```
Comment (lirona) [inline website/pages/blog/graphqlconf-2023-recap.mdx:98]:
```suggestion
in the GraphQL ecosystem, everyone plays a vital role in the success and future of GraphQL. The
```

File: website/pages/blog/graphqlconf-2023-recap.mdx [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+Lastly, the conference showcased several innovative projects that challenge the way we currently use
+GraphQL.
+[Grafast](https://graphql.org/conf/schedule/275443caa2eda5df06699b724efa533c/?name=The%20Future%20of%20Efficiency%20Is%20Here:%20Add%20Planning%20to%20Your%20Schema!),
+for instance, introduces a more efficient approach to implementing GraphQL schemas, while
+[Houdini](https://graphql.org/conf/schedule/ce430c038efa9a9c19743d1ccc702de9/?name=Houdini:%20A%20GraphQL-First%20Meta-Framework)
+and
+[Isograph](https://graphql.org/conf/schedule/2517f7a1d13ad3c0652e1b3cc5b65714/?name=Isograph%20%E2%80%94%20Rethink%20GraphQL-Powered%20React%20Apps)
+are exciting new GraphQL clients inspired by Relay, taking the GraphQL experience to new heights.
+These projects sparked lively discussions and left us with a sense of anticipation and plenty of new
+ideas to explore and improve the GraphQL community as a whole. And if we look very far aheads to the
```
Comment (lirona) [inline website/pages/blog/graphqlconf-2023-recap.mdx:138]:
```suggestion
ideas to explore and improve the GraphQL community as a whole. And if we look ahead to the
```

File: website/pages/blog/graphqlconf-2023-recap.mdx [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+future,
+[Aleksandra's talk about AI, GraphQL, and the Rise of Malleable Applications](https://graphql.org/conf/schedule/4feef977ceb883c69c91ccd2dd607aec/?name=AI,%20GraphQL,%20and%20the%20Rise%20of%20Malleable%20Applications)
+gave us a glimpse at what would be possible in the near future!
```
Comment (lirona) [inline website/pages/blog/graphqlconf-2023-recap.mdx:141]:
```suggestion
gave us a glimpse at the upcoming possibilities!
```

File: website/pages/blog/graphqlconf-2023-recap.mdx [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+ensuring that the knowledge and insights shared during the conference can reach a wider audience. We
+also invite you to join the discussions on the official [GraphQL discord](discord.graphql.org),
+where the vibrant community continues to thrive. Also, new community Meetup groups have been
+announced at the conference, so make sure to
+[signup here](https://linuxfoundation.surveymonkey.com/r/KWNWSM5) to attend, speak or organize your
+local Meetup.
+
+![The Guild's booth](/blog-assets/graphqlconf-2023-recap/booth.jpg)
+
+**The overwhelming response and attendance at the conference have made clear that GraphQL Conf will
```
Comment (lirona) [inline website/pages/blog/graphqlconf-2023-recap.mdx:156]:
```suggestion
**The overwhelming participation and turnout at the conference have made clear that GraphQL Conf will
```

File: website/pages/blog/graphqlconf-2023-recap.mdx [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+return in 2024.** So, if you missed out this year, there will be more opportunities to meet and
```
Comment (lirona) [inline website/pages/blog/graphqlconf-2023-recap.mdx:157]:
```suggestion
return in 2024.** So, if you missed out this year, rest assured there will be more opportunities to meet and
```

File: website/pages/blog/graphqlconf-2023-recap.mdx [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+architectural choices. For example, on the GraphQL Gateways subject, even with talks and
+announcements on great toolings for that type of architecture, it's important to note that schema
+composition, GraphQL gateways, and other tools should be adopted based on their practical benefits
+in specific workflows. The conference emphasized that these solutions should be embraced only if
+they genuinely enhance the efficiency and productivity of your company's engineering team. GraphQL
+Fusion
+([Conf Talk](https://graphql.org/conf/schedule/4a4e842d1cd0c06083f484d31225abd1/?name=GraphQL%20Fusion:%20Rethinking%20Distributed%20GraphQL)),
+for example, offers adaptability to any GraphQL setup, making it easier to migrate to a more
+distributed setup in the future and even support for federating APIs that are not GraphQL (OpenAPI,
+SOAP, gRPC, etc...). On the other hand, Apollo Federation requires a specific subgraph structure.
```
Comment (lirona) [inline website/pages/blog/graphqlconf-2023-recap.mdx:114]: maybe unnecessary to mention apollo here

File: website/pages/blog/graphqlconf-2023-recap.mdx [hunk 8]
```diff
@@ -0,0 +1,187 @@
...
+and making sure everyone knows that **our individual tools work separately and integrate with any
+other tool in the ecosystem**, including our competitors - **the opposite of a vendor lock-in**, so
+you can choose the best tool that works for you. But, if we need to provide a simple answer, without
+looking at your particular use case, our choice of tools and architecture is:
+
+- GraphQL Server - [Yoga Server](https://the-guild.dev/graphql/yoga-server) (see
+  [comparison to other servers](https://the-guild.dev/graphql/yoga-server/docs/comparison))
+- GraphQL Gateway - [GraphQL Mesh](https://the-guild.dev/graphql/mesh) (Supports Federation v1, v2,
+  Stitching and upcoming support for Fusion and subgraphs from any API Protocol like OpenAPI, gRPC,
+  etc...)
```
Comment (lirona) [inline website/pages/blog/graphqlconf-2023-recap.mdx:175]:
```suggestion
  etc.)
```

File: website/pages/blog/graphqlconf-2023-recap.mdx (hunk 7 above)
Comment (lirona) [inline website/pages/blog/graphqlconf-2023-recap.mdx:114]:
```suggestion
SOAP, gRPC, etc.). On the other hand, Apollo Federation requires a specific subgraph structure.
```

File: website/pages/blog/graphqlconf-2023-recap.mdx [hunk 9]
```diff
@@ -0,0 +1,187 @@
...
+
+The first day of the conference was workshops day, with many great practical workshops across
+everything GraphQL. You can find all the workshops and their recording in the schedule
+[here](https://graphql.org/conf/schedule/)
+
+The second day started with welcoming remarks by Lee, followed with great keynotes by Marc-Andre,
+Keith Babo, Theo Browne (would couldn't believe he was invited to give a keynote ;) ) , Bryan
+Robinson, and our very own Uri Goldshtein.
+
+[Uri's keynote on the state of GraphQL](<https://graphql.org/conf/schedule/e29bf518adeb99b2319fa8cb70d8f445/?name=Open%20Federation%20-%20Open%20Remote%20Schema%20Specifications%20with%20Other%20Specs%20(OpenAPI,%20etc.)>)
```
Comment (kamilkisiela) [inline website/pages/blog/graphqlconf-2023-recap.mdx:75]:
```suggestion
[Uri's keynote on the state of GraphQL](https://graphql.org/conf/schedule/e29bf518adeb99b2319fa8cb70d8f445/?name=Open%20Federation%20-%20Open%20Remote%20Schema%20Specifications%20with%20Other%20Specs%20(OpenAPI,%20etc.))
```

File: website/pages/blog/graphqlconf-2023-recap.mdx [hunk 10]
```diff
@@ -0,0 +1,187 @@
+---
+title: GraphQLConf 2023 Recap
+authors: [laurin, urigo]
```
Comment (kamilkisiela) [inline website/pages/blog/graphqlconf-2023-recap.mdx:3]:
```suggestion
authors: [laurin, uri]
```

**Pull request merged**